packages/form-builder/src/App.scss

```diff
@@ -347,3 +347,13 @@ svg {
 }
 
 /** ------------------------------- */
+
+/*
+Add top border to block card (block tab) to match panel body elements.
+This creates a consistent separator between the tabs and the rest of the sidebar.
+*/
+.block-editor-block-card {
+    border-top: 1px solid #e0e0e0;
+}
+
+/** ------------------------------- */
```

Comment:
@kjohnson I know this is super small but I noticed it's technically possible to select the block tab without a block selected which looks like this:

<img width="402" alt="Screen Shot 2022-12-22 at 4 26 36 PM" src="https://user-images.githubusercontent.com/10138447/209229872-7830636b-6552-4403-bf50-490c2b6d74d4.png">


So let's target the tabs themselves so even in those weird cases the border is still there lol


```suggestion

/*
Add border bottom to tabs, while removing body top border.
This creates a consistent separator between the tabs and the rest of the sidebar.
*/
.components-tab-panel__tabs {
    border-bottom: 1px solid #e0e0e0;
}

.components-panel__body {
    border-top: none;
}

/** ------------------------------- */
```